### PR TITLE
Asteroid Fixes and Improvements

### PR DESCRIPTION
**Pull request by AtnerNT**
### Changelog:
CL: [Improvement] Changed lavaland and asteroid ore generation. Lavaland now generates with larger quantities of bluespace, uranium and plasma. Asteroids now generate more metal, silver, gold and titanium.
CL: [Improvement] Dirt piles are now a prefab variant
CL: [Fix] Fixed asteroids having atmospheres.
CL: [Fix] Fixed ores not generating on asteroids
CL: [Fix] Fixed radars not updating location on shuttles.
CL: [Balance] Rebalanced Amber to be in line with other ore prices for its rarity. Sell price reduced from 7.5K per ore to 1.5K per ore. Buy prce reduced from 10K per ore to 5K per 2 ore.

### CODE REVIEW

**Comment by AtnerNT** [comment]
Gonna try squeeze in another fix gimme a bit

File: UnityProject/Assets/Scripts/Systems/Ore Generation/OreGenerator.cs [hunk 1]
```diff
@@ -64,19 +64,17 @@ public void RunOreGenerator()
...
 			{
 				weightedList.Add(ores);
 			}
 		}
-
 		//TODO move BoundsInt bounds = wallTilemap.cellBounds to metaTileMap
 		BoundsInt bounds = wallTilemap.cellBounds;
 		List<Vector3Int> miningTiles = new List<Vector3Int>();
-
+		Debug.Log(wallTilemap.cellBounds.ToString());
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Systems/Ore Generation/OreGenerator.cs]:
```suggestion
```

File: UnityProject/Assets/Scripts/Systems/Ore Generation/OreGenerator.cs [hunk 2]
```diff
@@ -92,6 +90,8 @@ public void RunOreGenerator()
 		}
 
 		int numberOfTiles = (int)((miningTiles.Count / 100f) * config.Density);
+		Debug.Log(numberOfTiles);
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Systems/Ore Generation/OreGenerator.cs]:
```suggestion
```

**Comment by AtnerNT** [comment]
Alright there we go, tests should hopefully all pass and then will be good for a merge

**Review by corp-0 — APPROVED** [review]

**Pull request merged**